src/pages/MenuPage.js

```diff
@@ -237,13 +237,14 @@ const mapDispatchToProps = dispatch => ({
 });
 
 const mapStateToProps = state => {
-  const { modules, user } = state;
+  const { modules, usingModules, user } = state;
   const { currentUser } = user;
-  const { usingDashboard, usingDispensary } = modules;
+  const { usingDashboard, usingDispensary, usingVaccines } = modules;
```

Comment:
I don't think `usingModules` is defined here.

The redux shape or the `state` variable would be equal to

```
{
    sync: {...}
    pages: {...}
    user: {..}
    navigation: {...}
    ....
    modules: {
        [whatever is returned from `ModuleReducer`,
         usingDashboard,
         usingDispensary,
         usingVaccines,
         usingModules,
    }
,
}
```

So 

```
const { modules, user } = state;
const { usingDashboard, usingDispensary, usingVaccines, usingModules } = modules;
```

Reply:
Makes sense, I missed that one before! thanks 😄